server/queries/basic-queries.js

```diff
@@ -0,0 +1,32 @@
+const User = require('../models/user-models');
+
+const getUsers = async (request, response) => {
+  try {
+    const users = await User.find();
+    response.status(200).send(users);
+  } catch (error) {
+    response.status(500);
```

Comment:
It's important to send the error message to the response in order to give context to the API client.
```suggestion
    response.status(500).send(error);
```